demos/imagenet_classification_demo/include/grid_mat.hpp

```diff
@@ -0,0 +1,116 @@
+// Copyright (C) 2018 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <algorithm>
+#include <set>
+#include <string>
+#include <vector>
+#include <queue>
+
+#include <opencv2/imgproc.hpp>
+#include <opencv2/core/core.hpp>
+
+class GridMat {
+public:
+    cv::Mat outimg;
+
+    explicit GridMat(const cv::Size maxDisp = cv::Size{1080, 1920},
+                     size_t rNum = 15, size_t cNum = 10, size_t rh = 60):
```

Comment:
`GridMat` should depend on a sample image size and the number of images that can be located on a whole grid. And it is up to `GridMat` how exactly the images will be placed, while `rNum` and `cNum` forces it to be 15x10 grid.